fastlane_core/lib/fastlane_core/device_manager.rb

```diff
@@ -202,17 +202,22 @@ def to_s
       def boot
         return unless is_simulator
         return unless os_type == "iOS"
+        return if self.state == 'Booted'
 
         UI.message("Booting #{self}")
-        `xcrun simctl boot #{self.udid}` unless self.state == "Booted"
+
+        `xcrun simctl boot #{self.udid} 2>/dev/null`
+        self.state = 'Booted'
```

Comment:
Maybe out of scope but I'd rather see this as symbols than strings, I think they stick better but I think that would imply a wider refactor.

Reply:
That would affect quite a bit. You'll see that when the [device instance is created](https://github.com/fastlane/fastlane/blob/master/fastlane_core/lib/fastlane_core/device_manager.rb#L54), it is using the grepped string from the `xcrun simctl` command to get the state.